### PR TITLE
458 rename credit class to credit type

### PR DESCRIPTION
**Pull request by maheshKRex**
renaming all occurences of Credit Class with Credit Type in code [](https://github.com/EmpowerPlastic/empowerchain/issues/458)

### CODE REVIEW

File: chain/proto/empowerchain/plasticcredit/events.proto [hunk 1]
```diff
@@ -41,15 +41,15 @@ message EventUpdateProject {
 // EventProjectApproved is an event emitted when a project is approved by the issuer associated with the projects credit class
```
Comment (ba1ciu) [inline chain/proto/empowerchain/plasticcredit/events.proto:41]: This one and some other comments also need refactoring.

File: chain/x/plasticcredit/keeper/msg_server_test.go [hunk 2]
```diff
@@ -424,6 +424,16 @@ func (s *TestSuite) TestUpdateApplicant() {
 			},
 			err: sdkerrors.ErrInvalidAddress,
 		},
+		"invalid name": {
```
Comment (ba1ciu) [inline chain/x/plasticcredit/keeper/msg_server_test.go:427]: Isn't this a part of message validation and not integration test?

File: chain/x/plasticcredit/keeper/msg_server_test.go [hunk 3]
```diff
@@ -1166,26 +1176,27 @@ func (s *TestSuite) TestIssueCredits() {
...
 				SerialNumber: "123",
 				CreditAmount: 1000,
+				MetadataUris: []string{"ipfs://CID1", "ipfs://CID2"},
 			},
 			expectedAmount: 0,
-			err:            plasticcredit.ErrIssuerNotAllowed,
+			err:            plasticcredit.ErrProjectNotFound,
 		},
-		"unexisting project": {
+		"empty serial number": {
```
Comment (ba1ciu) [inline chain/x/plasticcredit/keeper/msg_server_test.go:1190]: Same here? Message validation or should we leave it here?

File: chain/x/plasticcredit/simulation/operations.go [hunk 4]
```diff
@@ -422,7 +422,7 @@ func SimulateMsgCreateCreditClass(cdc *codec.ProtoCodec, ak plasticcredit.Accoun
 		res, _ := querier.Params(ctx, &plasticcredit.QueryParamsRequest{})
 		params := res.Params
 
-		spendable, neg := bk.SpendableCoins(sdkCtx, admin.Address).SafeSub(params.CreditClassCreationFee)
+		spendable, neg := bk.SpendableCoins(sdkCtx, admin.Address).SafeSub(params.CreditTypeCreationFee)
 		if neg {
 			return simtypes.NoOpMsg(plasticcredit.ModuleName, msgType, "not enough balance to create credit class"), nil, nil
```
Comment (ba1ciu) [inline chain/x/plasticcredit/simulation/operations.go:427]: Err message to refactor.

File: chain/x/plasticcredit/simulation/operations.go [hunk 5]
```diff
@@ -512,7 +512,7 @@ func SimulateMsgCreateProject(cdc *codec.ProtoCodec, ak plasticcredit.AccountKee
 			return simtypes.NoOpMsg(plasticcredit.ModuleName, msgType, "unable to get applicant"), nil, nil
 		}
 
-		creditClass, err := getRandomCreditClass(ctx, r, querier)
+		creditType, err := getRandomCreditType(ctx, r, querier)
 		if err != nil {
 			return simtypes.NoOpMsg(plasticcredit.ModuleName, msgType, "unable to get credit class"), nil, nil
```
Comment (ba1ciu) [inline chain/x/plasticcredit/simulation/operations.go:517]: Err message to refactor.

File: chain/x/plasticcredit/simulation/operations.go [hunk 6]
```diff
@@ -620,12 +620,12 @@ func SimulateMsgApproveProject(cdc *codec.ProtoCodec, ak plasticcredit.AccountKe
 			return simtypes.NoOpMsg(plasticcredit.ModuleName, msgType, "project already approved"), nil, nil
 		}
 
-		creditClass, found := querier.GetCreditClass(sdkCtx, project.CreditClassAbbreviation)
+		creditType, found := querier.GetCreditType(sdkCtx, project.CreditTypeAbbreviation)
 		if !found {
 			return simtypes.NoOpMsg(plasticcredit.ModuleName, msgType, "unable to get credit class"), nil, nil
```
Comment (ba1ciu) [inline chain/x/plasticcredit/simulation/operations.go:625]: Err message to refactor.

Comment (ba1ciu) [reply chain/x/plasticcredit/simulation/operations.go:625]: And some others below

**Review by ba1ciu — CHANGES_REQUESTED** [review]

File: chain/x/plasticcredit/keeper/msg_server_test.go (hunk 2 above)
Comment (maheshKRex) [reply chain/x/plasticcredit/keeper/msg_server_test.go]: yeah, i messed up while resolving merge conflicts with too many files :) , removed all the added back tests there.

File: chain/x/plasticcredit/simulation/operations.go (hunk 5 above)
Comment (maheshKRex) [reply chain/x/plasticcredit/simulation/operations.go]: updated Err message

File: chain/x/plasticcredit/simulation/operations.go (hunk 6 above)
Comment (maheshKRex) [reply chain/x/plasticcredit/simulation/operations.go]: Updated err messages

**Review by ba1ciu — APPROVED** [review]

**Pull request merged**